src/api/localStorage.js

```diff
@@ -0,0 +1,90 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('UserAccounts')) || [];
+  }
+
+  createUser(user) {
+    this.users.push(JSON.stringify(user));
```

Comment:
as discussed in slack, you'll want to just push the user object **without** stringifying it.
```suggestion
    this.users.push(user);
```

you'll still want to stringify `this.users` when setting `UserAccounts`, though. just don't need the double `stringify`.